### PR TITLE
0.28.0 RPM Without Mesos

### PR DESCRIPTION
**Pull request by ozdanborne**

### CODE REVIEW

File: packages/net-modules.spec [hunk 1]
```diff
@@ -32,13 +29,15 @@ The first implementation in this repository showcases Apache Mesos using Project
...
 ./bootstrap
 
-%configure --with-mesos=/usr/include/mesos/ --with-protobuf=/usr
+CPPFLAGS='-I/root/protobuf/protobuf-2.5.0/src/ -I/root/glog/glog-0.3.3/src/ -I/root/boost/boost-1.53.0/' ./configure --with-mesos=/
 make -j 2
 
 %install
 %make_install
 
+rm -f /usr/local/lib/mesos/*.la
```
Comment (karya0) [inline packages/net-modules.spec]: I don't understand this line and also line # 40. What are these files that we are removing?

File: packages/net-modules.spec [hunk 2]
```diff
@@ -46,7 +45,7 @@ install %{SOURCE2} %{buildroot}%{_sysconfdir}/mesos-slave/
 
 ############################################
 %files
-%{_libdir}/mesos/libmesos_network_isolator*.so
+/usr/local/lib/mesos/libmesos_network_isolator*.so
```
Comment (karya0) [inline packages/net-modules.spec]: Why stop using the `_libdir` macro?

File: packages/Dockerfile [hunk 3]
```diff
@@ -39,15 +35,10 @@ RUN yum install -y \
   libev-source \
   leveldb-devel \
   protobuf-java \
```
Comment (karya0) [inline packages/Dockerfile]: remove this too?

File: packages/net-modules.spec (hunk 1 above)
Comment (djosborne) [inline packages/net-modules.spec]: [Standard RPM packaging rules say to remove all static library files](https://fedoraproject.org/wiki/Packaging:Guidelines). I copied this from [our original mesos spec](https://github.com/mesosphere/net-modules/pull/109/files#diff-b6afc46629fd72a7ea1541f1a3ba0dbcL168)

File: packages/net-modules.spec (hunk 2 above)
Comment (djosborne) [inline packages/net-modules.spec]: The RPM builder resolves `%{_libdir}` to `/usr/lib64/`, but net-modules compilation puts the files in `/usr/lib`. So you end up seeing this error:
```
RPM build errors:
File not found by glob: /root/rpmbuild/BUILDROOT/net-modules-0.2.0-1.custom.x86_64/usr/lib64/mesos/libmesos_network_isolator*.so
```

I'm not sure if its net-modules or the rpm builder that needs to be reconfigured. Any thoughts @karya0 ?

Comment (djosborne) [inline packages/net-modules.spec]: Got it. Changed the install directory to be /usr, no more local.

File: packages/net-modules.spec (hunk 1 above)
Comment (djosborne) [inline packages/net-modules.spec]: Fixed this - removed the duplicate lines

**Pull request merged**